### DISCUSSION TITLE
Getting requirements to build wheel ... error   error: subprocess-exited-with-error

### DISCUSSION THREAD

**Discussion by RAM7RKC** [SciTools/iris#5480] [Q&A]
Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [20 lines of output]
      running egg_info
      writing src\antlr4_python3_runtime.egg-info\PKG-INFO
      writing dependency_links to src\antlr4_python3_runtime.egg-info\dependency_links.txt
      writing requirements to src\antlr4_python3_runtime.egg-info\requires.txt
      writing top-level names to src\antlr4_python3_runtime.egg-info\top_level.txt
      reading manifest file 'src\antlr4_python3_runtime.egg-info\SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      writing manifest file 'src\antlr4_python3_runtime.egg-info\SOURCES.txt'
      OSError: [Errno 9] Bad file descriptor

      During handling of the above exception, another exception occurred:

      Traceback (most recent call last):
        File "C:\Users\RAM\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "C:\Users\RAM\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 349, in main
          write_json(json_out, pjoin(control_dir, 'output.json'), indent=2)
        File "C:\Users\RAM\AppData\Local\Programs\Python\Python311\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 31, in write_json
          with open(path, 'w', encoding='utf-8') as f:
      OSError: [Errno 9] Bad file descriptor
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

**Comment by rcomer** [comment]
This is likely related to https://github.com/SciTools/cf-units/issues/313
